meetings/2025-08-19-taskforce-meeting.md

```diff
@@ -0,0 +1,62 @@
+# 2025-08-19: ActivityPub Trust and Safety Task Force Meeting
+
+## Attendees:
+- Emelia Smith <@thisismissem@hachyderm.io>
+- Evan Prodromou <web+acct:evanprodromou@socialwebfoundation.org>
+- amy bones <@amy@spookygirl.boo>
+
+## Agenda:
+
+1. IP Protection Note Reminder:
+    - Anyone can participate in these calls. However, all substantive contributors to any Taskforce Work Items must be members of the Social Web CG with full IPR agreements signed. https://www.w3.org/community/socialcg/join
+    - To contribute to Work Items: ensure you have a W3 account: https://www.w3.org/accounts/request, and sign the W3C Community Contributor License Agreement (CLA): https://www.w3.org/community/about/agreements/cla/
+2. Reminder of [Code of Conduct](https://github.com/swicg/activitypub-trust-and-safety/blob/main/CODE_OF_CONDUCT.md)
+3. Introductions, if necessary
+4. New meeting time - https://github.com/swicg/activitypub-trust-and-safety/issues/91
+    - Possible options:
+        - 10:00–11:00 Eastern Daylight Time (16:00-17:00 CEST), on Wednesday's every month
+        - 11:00–12:00 Eastern Daylight Time (17:00-18:00 CEST), on Wednesday's every month
+        - 12:00–13:00 Eastern Daylight Time (18:00-19:00 CEST), on Wednesday's every month
+
+6. Funding / Grants?
+~~7. FEP updates~~
+    - ~~Labels~~
+    - ~~Content Warnings~~
+    - ~~Moderation Actor (new?)~~
+~~9. Open floor~~
+
+We support contributions to the Agenda, please comment if there is something that we need to address during the meeting.
+
+You can find information on how to join the meeting in the [SWICG calendar](https://www.w3.org/events/meetings/a54ae3c9-89bc-4bb1-b9db-e9494d2100e1/20250121T110000/)
+
+### Previous Action Items
+
+- [x] Evan: add an issue to expand the scope of as:sensitive to Link as well as Object
+    - PR is also submitted to make this change. 
+- [ ] Darius: talk to w3c about funded task force work in general and report back
```

Comment:
@dariusk I'm not sure we need follow up from you on this, unless we learned anything specific from W3C? This is a community group taskforce, so as long as IPR is fine, I think we should be fine. We may need a logo in the README for the funder, but that should be fine too.